src/cms/templates/pages/page_form.html

```diff
@@ -161,7 +174,18 @@ <h2 class="heading font-normal">
                         <a href="{{ page_translation_form.instance.short_url }}" target="_blank" rel="noopener noreferrer" class="text-blue-500 hover:underline">{{ page_translation_form.instance.short_url }}</a>
                         <a href="#" onclick="short_url_to_clipboard('{{ page_translation_form.instance.short_url }}');" title="{% trans 'Copy to clipboard' %}" class="px-2 text-gray-800 hover:text-blue-500">
                             <i data-feather="copy" class="inline-block"></i>
-                        </a>
+                        </a><br>
+                        {% if page_translation_form.instance.currently_in_translation %}
+                            <i data-feather="alert-triangle" class="inline-block text-red-800"></i>
+                            <label class="inline-block mb-2 font-bold">{% trans 'Warning' %}:</label>
+                            {% trans 'The page is currently in translation for this language' %}
```

Comment:
What about a simple `Translation in progress` as an alternative to `The page is currently in translation for this language`? Would be shorter and I think it conveys the same information.